<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
          <meta charset="UTF-8">
          <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
          <meta http-equiv="x-ua-compatible" content="ie=edge">

        <meta http-equiv="content-type" content="text/html; charset=utf-8" />
        <meta name="description" content="" />
        <meta name="keywords" content="" />
        <title>Xiao Wang</title>
        <link rel="stylesheet" type="text/css" href="css/style.css" />


          <link rel="stylesheet" href="css/bootstrap.min.css">
          <link rel="stylesheet" href="css/academicons.min.css">
          <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
          <link rel="stylesheet" href="css/font-awesome.min.css">


    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css" integrity="sha256-+N4/V/SbAFiW1MPBCXnfnP9QSN3+Keu+NlB+0ev/YKQ=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Libre+Baskerville%7CMontserrat:400,700%7CRoboto%7CRoboto Mono&display=swap">
    <link rel="stylesheet" href="/css/academic.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/github.min.css" crossorigin="anonymous" title="hl-light">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">

            <style>
            a.plain-black {
              text-decoration: none;
              color: black;
              margin: 5px;
            }
          </style>



    </head>
    <body>
    


        <div id="wrapper">

            <div id="inner">
                <div id="page">
                    <div id="content">
                            <table>
                            <tr>
                                <td align="left" width = "180"><img class="image" src="image/Xiao_Wang2023.jpeg" width = "180px" alt="Xiao Wang" /></td>                                <td>
                                <h2>Xiao Wang</h2>
                                <p>Research Scientist@<a href="https://uw.edu" target="_blank">TikTok</a> <br>
                                CS Ph.D.@<a href="https://purdue.edu" target="_blank">Purdue University</a> <br>
                                    E-mail: wang3702 [at] uw [dot] edu <br>
                                    <a class="plain-black" href="mailto:wang3702@purdue.edu"><i class="fa fa-2x fa-envelope"></i></a>
									<a class="plain-black" href="https://scholar.google.com/citations?user=AGS_dK8AAAAJ&hl=en" target="_blank"><i class="ai ai-google-scholar-square ai-2x"></i></a>
                                    <a class="plain-black" href="https://orcid.org/0000-0003-4435-7098" target="_blank"><i class="ai ai-orcid-square ai-2x"></i></a>
                                     <a class="plain-black" href="https://www.researchgate.net/profile/Xiao-Wang-63" target="_blank"><i class="ai ai-researchgate-square ai-2x"></i></a>
                                    <a class="plain-black" href="https://github.com/wang3702/wang3702.github.io/blob/master/files/CV.pdf" target="_blank"><i class="fa fa-2x fa-book"></i></a>
                                </p>
                                 </td>
                                <td>
                                <td align="right" bgcolor="" width = "500">
   <iframe src="https://kiharalab.org/dove/xiaowang/php_scholar/pub.php" onload='javascript:(function(o){o.style.height=o.contentWindow.document.body.scrollHeight+"px";}(this));' style="height:300px;width:100%;border:none;overflow:hidden;"></iframe>
                                </td>
                            </tr>


                            </table>
                            <br/>
                            <h3>About Me</h3>
                            <table>
                            <tr>
                                <td align="justify" bgcolor="" width = "1000px",  line-height = "120px" >
                                <p>
                                   Xiao Wang is currently a research scientist at TikTok, focusing on multimodal learning.
                                   Previously, he was a postdoc at University of Washington, working on AI for biology. 
                                   Prior to that, he obtained a CS Ph.D. degree from <a href="https://cs.purdue.edu" target="_blank">Purdue University</a>, focusing on representation learning and AI for biology.
                                   He graduated with a bachelor's degree in CS from <a href="http://en.xjtu.edu.cn/" target="_blank">Xi'an Jiaotong University</a>, Xi'an, China.
                                </p>
                                <p>
                                    He used interned in Futurewei AI lab, Jingdong AI research, Meta AI (FAIR Labs).  
                                    His research interests include multi-modal learning,AI for biology, representation learning and generative modeling.
                                </p>
                                </td>
                              </tr>


                            </table>
<!--      <p style="line-height:1.4; margin:1em; width: 80%"> <b>-->
<!--<div style="margin:1em;line-height:1.5;color:#800000;background-color:white;border-color:#c3e6cb;padding:5px;border:1px dashed;width:85%">-->
<!--<i class="fa fa-bullhorn" style="font-size:36px;color:brick"></i>-->
<!--   I'm currently actively looking for academic positions (Faculty+Postdoc).-->
<!--Please contact me if there are any available positions.-->
<!--</div>-->
<!--</p>-->

                        <h3>Fellowship&Awards</h3>

                        <table>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Jan 2025 - June 2025</b>: <a href = "https://www.nvidia.com/en-us/industries/higher-education-research/academic-grant-program/" target="_blank">NVIDIA Academic GPU Grant (15,000 hour A100 credits)</a>
                                    </li>
                                </td>
                              </tr>
                            <tr>

                                <tr>
                                    <td align="left" bgcolor="" width = "1000px" >
                                        <li> <b>Jan 2025 - Dec 2025</b>: Microsoft Azure GPU Grant ($20,000)</a>
                                        </li>
                                    </td>
                                  </tr>
                                <tr>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Jan 2024 - Dec 2024</b>: <a href = "https://escience.washington.edu/people/postdoctoral-fellows/" target="_blank">UW Data Science Postdoctoral Fellow</a>
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>July 2022 - June 2023</b>: <a href = "https://molssi.org" target="_blank">NSF MolSSI Fellowship</a> with $80,000 for stipend, tuition and fees.
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Aug 2018 - July 2019</b>: <a href = "http://www.ccicf.org.hk/en/scholarships/fellowship_b_04/" target="_blank">Chiang Chen Overseas Fellowship</a> with $50,000 for tuition and fees.
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Sep 2017 - July 2018</b>: HIWIN Outstanding Student Scholarship with ￥10,000 for tuition.
                                    </li>

                                </td>
                              </tr>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Dec 2017</b>: Top 10 Undergraduate of Xi'an Jiaotong University.
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Sep 2016 - July 2017</b>: National Scholarship with ￥8,000 for tuition.
                                    </li>

                                </td>
                              </tr>

                        </table>

                            <!-- <h3>Recent News</h3>
                        <table>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Jul 2024</b>:  <a href = "https://www.nature.com/articles/s41592-024-02479-0" target="_blank">DiffModeler</a>  got accepted in <font color="red">Nature Methods</font>.
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Jan 2024</b>:  <a href = "https://www.nature.com/nmeth/volumes/21/issues/1" target="_blank">DeepMainMast</a> is selected as the cover of Volume 21 issue 1 of <font color="red">Nature Methods</font>.
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Sep 2023</b>:  <a href = "https://www.nature.com/articles/s41592-023-02099-0" target="_blank">DeepMainMast</a> got accepted in <font color="red">Nature Methods</font>.
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>May 2023</b>: <a href = "https://www.nature.com/articles/s41592-023-02032-5" target="_blank">CryoREAD</a> got accepted in <font color="red">Nature Methods</font> and selected for <a href="https://www.nature.com/articles/s41592-023-02033-4" target="_blank">research briefing</a>.
                                    </li>

                                </td>
                              </tr>
                             <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>March 2023</b>: <a href = "https://doi.org/10.1038/s41592-023-01876-1" target="_blank">DAQ-Database</a> got accepted in <font color="red">Nature Methods</font>.
                                    </li>

                                </td>
                              </tr>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>March 2023</b>: <a href = "https://doi.org/10.1109/TPAMI.2023.3262608" target="_blank">CaCo</a> got accepted in IEEE Transactions on Pattern Analysis and Machine Intelligence (<font color="red">IEEE T-PAMI</font>).
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>March 2023</b>: <a href = "https://doi.org/10.1109/TNNLS.2023.3263387" target="_blank">CoSeg</a> got accepted in IEEE Transactions on Neural Networks and Learning Systems (<font color="red">IEEE TNNLS</font>).
                                    </li>

                                </td>
                              </tr> -->
<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>March 2023</b>: I was invited for initial screening in Computer Science Department of <a href = "https://cs.unc.edu/" target="_blank">The University of North Carolina at Chapel Hill</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->
<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Jan 2023</b>: I was invited for initial screening in Computer Science Department of <a href = "https://cs.brown.edu/" target="_blank">Brown University</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Dec 2022</b>: I was invited for initial screening in Computer Science Department of <a href = "https://cs.vt.edu/" target="_blank">Virginia Tech (VT)</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Nov 2022</b>: I was selected as Associate Editor (AE) of  <a href = "https://ieeexplore.ieee.org/xpl/aboutJournal.jsp?punumber=7274857" target="_blank">IEEE Transactions on Intelligent Vehicles (T-IV)</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                             <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Nov 2022</b>: I was invited to give a talk for Nanyang Professorship of <a href = "https://www.ntu.edu.sg/research/research-careers/nanyang-assistant-professorship-(nap)" target="_blank">Nanyang Technological University (NTU)</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Nov 2022</b>: I was invited for initial screening in Computer Science&Engineering Department of <a href = "https://cse.nd.edu/" target="_blank">The University of Notre Dame (ND)</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Nov 2022</b>: I was invited to give a seminar in Computer Science Department of <a href = "https://www.cs.hku.hk/" target="_blank">The University of Hong Kong (HKU)</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Oct 2022</b>: I was invited to give a talk in Computer Science Department of <a href = "https://cemse.kaust.edu.sa/cs" target="_blank">King Abdullah University of Science and Technology (KAUST)</a>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

                            <!-- <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>July 2022</b>: My old paper <a href = "https://ieeexplore.ieee.org/document/7970189" target="_blank">GRU-CF</a> was nominated for <a href = "https://ieee-itss.org/pub/t-its/t-its-paper-award-history/" target="_blank">George N. Saridis Best Transactions Paper Award</a>.
                                    </li>

                                </td>
                              </tr>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>June 2022</b>: <a href = "https://www.nature.com/articles/s41592-022-01574-4" target="_blank">DAQ</a> got accepted in <font color="red">Nature Methods</font>.
                                    </li>

                                </td>
                              </tr>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>May 2022</b>: I got <a href = "https://molssi.org" target="_blank">NSF MolSSI Fellowship</a> by MolSSI of National Science Foundation (NSF).
                                    </li>

                                </td>
                              </tr> -->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>April 2022</b>: <a href = "https://doi.org/10.1109/TPAMI.2022.3203630" target="_blank">CLSA</a> got accepted in IEEE Transactions on Pattern Analysis and Machine Intelligence (<font color="red">IEEE T-PAMI</font>).-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->
<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>March 2022</b>: <a href = "https://openaccess.thecvf.com/content/CVPR2022/papers/Wang_On_the_Importance_of_Asymmetry_for_Siamese_Representation_Learning_CVPR_2022_paper.pdf" target="_blank">Asym-Siam</a> got accepted in <font color="red">CVPR 2022</font>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->
<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Sep 2021</b>: <a href = "https://www.sciencedirect.com/science/article/pii/S2589004221009858" target="_blank">Attention_AD</a> got accepted in <b>iScience</b>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->
<!--&lt;!&ndash;                            <tr>&ndash;&gt;-->
<!--&lt;!&ndash;                                <td align="left" bgcolor="" width = "1000px" >&ndash;&gt;-->
<!--&lt;!&ndash;                                    <li> <b>Aug 2021</b>: <a href = "https://doi.org/10.3389/fmolb.2021.647915" target="_blank">GNN-DOVE</a> got accepted in <b>Frontiers in Molecular Biosciences (FMOLB)</b>.&ndash;&gt;-->
<!--&lt;!&ndash;                                    </li>&ndash;&gt;-->

<!--&lt;!&ndash;                                </td>&ndash;&gt;-->
<!--&lt;!&ndash;                              </tr>&ndash;&gt;-->


<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>April 2021</b>: <a href = "https://www.nature.com/articles/s41467-021-22577-3" target="_blank">Emap2sec+</a> got accepted in <font color="red">Nature Communications</font>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>March 2021</b>: <a href = "https://openaccess.thecvf.com/content/CVPR2021/papers/Hu_AdCo_Adversarial_Contrast_for_Efficient_Learning_of_Unsupervised_Representations_From_CVPR_2021_paper.pdf" target="_blank">AdCo</a> got accepted in <font color="red">CVPR 2021</font>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->
<!--                             <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Dec 2020</b>: <a href = "https://ieeexplore.ieee.org/document/9301261" target="_blank">EnAET</a> got accepted in IEEE Transcations on Image Processing (<font color="red">IEEE-TIP</font>).-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->

<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>Oct 2020</b>: <a href = "https://doi.org/10.1109/TPAMI.2020.3029801" target="_blank">AET</a> got accepted in IEEE Transactions on Pattern Analysis and Machine Intelligence (<font color="red">IEEE T-PAMI</font>).-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->
<!--                            <tr>-->
<!--                                <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li> <b>April 2020</b>: <a href = "https://academic.oup.com/bioinformatics/article/36/7/2113/5634144" target="_blank">DOVE</a> got accepted in <b>Bioinformatics</b>.-->
<!--                                    </li>-->

<!--                                </td>-->
<!--                              </tr>-->
                        <!-- </table> -->



            <h3>Selected publications</h3>
            * denotes equal contribution
            
            <h4>2025</h4>
            <! –– Evo2HiC ––>
            <table>
                <tr>
                    <td align="left" width = "50px"></td>
                    <td align="left" width = "350px">
                        <img class="image" src="image/evo2hic.jpg" width = "250px" alt="" />

                    </td>
                    <td align="left" bgcolor="" width = "600px" >
                        <b>Evo2HiC: a multimodal foundation model for integrative analysis of genome sequence and architecture</b> <br/>
                        Tangqi Fang* <font color="blue">Xiao Wang*</font>, Zhiping Xiao, Shengqi Hang, Ghulam Murtaza, Junwei Yang, Hanwen Xu, Anupama Jha, William Noble, Sheng Wang<br/>
                        <i><font color="red">Nature Genetics (under review)</font></i>, 2025<br/>
                        <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/2025.11.18.689171v1" target="_blank" rel="noopener">
                            <i class="fas fa-file-pdf mr-1"></i> Paper
                        </a>
                        <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/CHNFTQ/Evo2HiC" target="_blank" rel="noopener">
                        <i class="fas fa-code mr-1"></i>Code
                        </a>
                        <br/>
                        <font color="#FF9966">[AI for biology]</font>,  <font color="#3399FF">[Representation learning]</font>
                    </td>
                  </tr>
                </table>

            <! –– HiCFoundation ––>
            <table>
                <tr>
                    <td align="left" width = "50px"></td>
                    <td align="left" width = "350px">
                        <img class="image" src="image/hicfoundation_logo.png" width = "250px" alt="" />

                    </td>
                    <td align="left" bgcolor="" width = "600px" >
                        <b>A generalizable foundation model for chromatin architecture, single-cell and multi-omics analysis across species</b> <br/>
                        <font color="blue">Xiao Wang*</font>, Yuanyuan Zhang*, Suhita Ray, Anupama Jha, Tangqi Fang, Shengqi Hang, Sergei Doulatov, William Stafford Noble, Sheng Wang<br/>
                        <i><font color="red">Nature Methods (accepted)</font></i>, 2025<br/>
                        <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/2024.12.16.628821v1" target="_blank" rel="noopener">
                            <i class="fas fa-file-pdf mr-1"></i> Paper
                        </a>
                        <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/Noble-Lab/HiCFoundation" target="_blank" rel="noopener">
                        <i class="fas fa-code mr-1"></i>Code
                        </a>
                        <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://colab.research.google.com/github/Noble-Lab/HiCFoundation/blob/main/HiCFoundation.ipynb" target="_blank" rel="noopener">
                        <i class="fas fa-code mr-1"></i>Colab
                        </a>
                        <br/>
                        <font color="#FF9966">[AI for biology]</font>, <font color="#CC3366">[Generative modeling]</font>, <font color="#3399FF">[Representation learning]</font>
                    </td>
                  </tr>
                </table>

                <! –– CryoNeRF ––>
                <table>
                    <tr>
                        <td align="left" width = "50px"></td>
                        <td align="left" width = "350px">
                            <img class="image" src="image/cryonerf.png" width = "250px" alt="" />
    
                        </td>
                        <td align="left" bgcolor="" width = "600px" >
                            <b>CryoNeRF: generalizable automated cryo-EM reconstruction using neural radiance field</b> <br/>
                            Huaizhi Qu*, <font color="blue">Xiao Wang*</font>, Yuanyuan Zhang, Sheng Wang, William Stafford Noble, Tianlong Chen<br/>
                            <i>BioRxiv</i>, 2025<br/>
                            <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/2025.01.10.632460v1" target="_blank" rel="noopener">
                                <i class="fas fa-file-pdf mr-1"></i> Paper
                            </a>
                            <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/UNITES-Lab/CryoNeRF" target="_blank" rel="noopener">
                            <i class="fas fa-code mr-1"></i>Code
                            </a>
                            <br/>
                            <font color="#FF9966">[AI for biology]</font>,  <font color="#3399FF">[Representation learning]</font>
                        </td>
                      </tr>
                    </table>

                    <! –– FlowTS––>
                    <table>
                        <tr>
                            <td align="left" width = "50px"></td>
                            <td align="left" width = "350px">
                                <img class="image" src="image/flowts.png" width = "250px" alt="" />
        
                            </td>
                            <td align="left" bgcolor="" width = "600px" >
                                <b>FlowTS: Time Series Generation via Rectified Flow</b> <br/>
                                Yang Hu*, <font color="blue">Xiao Wang*</font>,  Zezhen Ding, Lirong Wu, Huatian Zhang, Stan Z. Li, Sheng Wang, Jiheng Zhang, Ziyun Li, Tianlong Chen<br/>
                                <i>Arxiv</i>, 2025<br/>
                                <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://arxiv.org/abs/2411.07506" target="_blank" rel="noopener">
                                    <i class="fas fa-file-pdf mr-1"></i> Paper
                                </a>
                                <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/UNITES-Lab/FlowTS" target="_blank" rel="noopener">
                                <i class="fas fa-code mr-1"></i>Code
                                </a>
                                <br/>
                                <font color="#CC3366">[Generative modeling]</font>
                            </td>
                          </tr>
                        </table>
        
                        <!-- <h4>AI for biology</h4> -->
                <h4>2024</h4>     
                        <! –– DiffModeler ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/diffmodeler.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>DiffModeler: Large Macromolecular Structure Modeling for Cryo-EM Maps Using Diffusion Model</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Han Zhu, Genki Terash, Manav Taluja, Daisuke Kihara<br/>
                                    <i><font color="red">Nature Methods</font></i>, 2024<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41592-024-02479-0" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/DiffModeler" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://em.kiharalab.org/algorithm/DiffModeler" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Server
                                    </a>
                                    <br/>
                                    <font color="#FF9966">[AI for biology]</font>, <font color="#CC3366">[Generative modeling]</font>
                                </td>
                              </tr>
                            </table>

                            <! –– DeepMainMast ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/dmm_cover.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>DeepMainmast: integrated protocol of protein structure modeling for cryo-EM with deep learning and structure prediction</b> <br/>
                                     Genki Terash,  <font color="blue">Xiao Wang</font>, Devashish Prasad, Tsukasa Nakamura, Daisuke Kihara<br/>
                                    <i><font color="red">Nature Methods</font></i>, 2024<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41592-023-02099-0" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/DeepMainMast" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://colab.research.google.com/github/kiharalab/DeepMainMast/blob/main/DeepMainMast.ipynb" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Colab
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://em.kiharalab.org/algorithm/DeepMainMast" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>Server
                                        </a>
                                    <br/>
                                    Selected as the cover of Volume 21 issue 1 of <a href="https://www.nature.com/nmeth/volumes/21/issues/1" target="_blank">Nature Methods</a>
                                    <font color="#FF9966">[AI for biology]</font>
                                </td>
                              </tr>
                            </table>
                    

                <h4>2023</h4>           

                        <! –– CryoREAD ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/cryoread_logo.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>De Novo Structure Modeling for Nucleic Acids in cryo-EM Maps Using Deep Learning</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Genki Terash,  Daisuke Kihara<br/>
                                    <i><font color="red">Nature Methods</font></i>, 2023<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41592-023-02032-5" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/CryoREAD" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://bit.ly/CryoREAD" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Colab
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://em.kiharalab.org/algorithm/CryoREAD" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>Server
                                        </a>
                                    <br/>
                                    Selected for research briefing <a href="https://www.nature.com/articles/s41592-023-02033-4" target="_blank">CryoREAD provides fully automated DNA–RNA structure modeling for cryo-EM maps</a> in Nature Methods.
                                    <font color="#FF9966">[AI for biology]</font>
                                </td>
                              </tr>
                            </table>
                         

                        <! –– DAQ DB ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/daqdb.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>DAQ-Score Database: Assessment of Map-Model Compatibility for Protein Structure Models from Cryo-EM Maps</b> <br/>
                                    Tsukasa Nakamura, <font color="blue">Xiao Wang</font>, Genki Terash,  Daisuke Kihara<br/>
                                    <i><font color="red">Nature Methods</font></i>, 2023 <br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41592-023-01876-1" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://daqdb.kiharalab.org" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Database Server
                                    </a>
                                    <br/>
                                    <font color="#FF9966">[AI for biology]</font>
                                </td>
                              </tr>
                            </table>



                            <! –– CaCo ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/caco.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>CaCo: Both Positive and Negative Samples are Directly Learnable via Cooperative-adversarial Contrastive Learning</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Yuhang Huang, Dan Zeng, Guo-Jun Qi<br/>
                                    <i>IEEE Transactions on Pattern Analysis and Machine Intelligence (<font color="red">IEEE T-PAMI</font>), </i>2023<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1109/TPAMI.2023.3262608" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/maple-research-lab/caco" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <br/>
                                    <font color="#3399FF">[Representation learning]</font>
                                </td>
                              </tr>
                            </table>

                    <h4>2022</h4>          

                            <! –– DAQ ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/daq_logo.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>Residue-Wise Local Quality Estimation for Protein Models from Cryo-EM Maps</b> <br/>
                                     Genki Terash*, <font color="blue">Xiao Wang*</font>, Sai Raghavendra Maddhuri Venkata Subramaniya, John J. G. Tesmer, Daisuke Kihara<br/>
                                    <i><font color="red">Nature Methods</font></i>, 2022 <br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41592-022-01574-4" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/DAQ" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://bit.ly/daq-score" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Colab
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://em.kiharalab.org/algorithm/daqscore" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>Server
                                        </a>
                                    <br/>
                                    <font color="#FF9966">[AI for biology]</font>
                                </td>
                              </tr>
                            </table>

                            <! –– Asym ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/asym.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>On the Importance of Asymmetry for Siamese Representation Learning</b> <br/>
                                    <font color="blue">Xiao Wang*</font>, Haoqi Fan*, Yuandong Tian, Daisuke Kihara, Xinlei Chen<br/>
                                    <i>Conference on Computer Vision and Pattern Recognition (<font color="red">CVPR</font>)</i>, 2022<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://openaccess.thecvf.com/content/CVPR2022/papers/Wang_On_the_Importance_of_Asymmetry_for_Siamese_Representation_Learning_CVPR_2022_paper.pdf" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/facebookresearch/asym-siam" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <br/>
                                    <font color="#3399FF">[Representation learning]</font>
                                </td>
                              </tr>
                            </table>

                            <! –– CLSA ––>
                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px"><img class="image" src="image/CLSA.jpeg" width = "250px" alt="" /></td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>Contrastive Learning with Stronger Augmentation</b> <br/>
                                     <font color="blue">Xiao Wang</font>, Guo-Jun Qi <br/>
                                    <i>IEEE Transactions on Pattern Analysis and Machine Intelligence (<font color="red">IEEE T-PAMI</font>)</i>, 2022<br/>
                                     <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1109/TPAMI.2022.3203630" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                     <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/maple-research-lab/clsa" target="_blank" rel="noopener">
                                         <i class="fas fa-code mr-1"></i>Code
                                     </a>
                                    <br/>
                                    <font color="#3399FF">[Representation learning]</font>
                                </td>
                              </tr>
                            </table>

                            <! –– CoSeg ––>
                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px"><img class="image" src="image/coseg.png" width = "250px" alt="" /></td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>CoSeg: Cognitively Inspired Unsupervised Generic Event Segmentation</b> <br/>
                                     <font color="blue">Xiao Wang</font>, Jingen Liu, Tao Mei, Jiebo Luo <br/>
                                    <i>IEEE Transactions on  Neural Networks AND Learning Systems (<font color="red">IEEE TNNLS</font>)</i>, 2022<br/>
                                     <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1109/TNNLS.2023.3263387" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/wang3702/CoSeg" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>
                                        code
                                    </a>
                                    <br/>
                                    <font color="#3399FF">[Representation learning]</font>
                                </td>
                              </tr>
                            </table>
                         <!-- <! –– DAQ-Refine ––>
                        <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/daq_refine.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>Protein Model Refinement for Cryo-EM Maps Using DAQ score</b> <br/>
                                    Genki Terashi, <font color="blue">Xiao Wang</font>, Daisuke Kihara<br/>
                                    <i>Acta Crystallographica Section D:  Structural Biology</i>, 2022 <br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1107/S2059798322011676" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/DAQ-Refine" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://bit.ly/DAQ-Refine" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Colab
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://em.kiharalab.org/algorithm/daq-refine" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>Server
                                        </a>
                                    <br/>
                                </td>
                              </tr>
                            </table> -->
                <h4>2021</h4>  
                        <! –– emap2secplus––>
                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px"><img class="image" src="image/emap2secplus.png" width = "250px" alt="" /></td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>Detecting Protein and DNA/RNA Structures in Cryo-EM Maps of Intermediate Resolution Using Deep Learning</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Eman Alnabati, Tunde W Aderinwale, Sai Raghavendra Maddhuri, Genki Terashi, Daisuke Kihara <br/>
                                    <i><font color="red">Nature Communications</font></i>, 2021<br/>
                                     <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41467-021-22577-3" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                      <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/Emap2secPlus" target="_blank" rel="noopener">
                                          <i class="fas fa-code mr-1"></i>Code
                                      </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.24433/CO.7165707.v1" target="_blank" rel="noopener">
                                          <i class="fas fa-code mr-1"></i>Code_Ocean
                                      </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://colab.research.google.com/github/kiharalab/Emap2secPlus/blob/master/Emap2sec%2B.ipynb" target="_blank" rel="noopener">
                                          <i class="fas fa-code mr-1"></i>Colab
                                      </a>
                                      <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://em.kiharalab.org/algorithm/emap2sec+" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>Server
                                        </a>
                                    <br/>
                                    <font color="#FF9966">[AI for biology]</font>
                                </td>
                              </tr>
                            </table>
                            <! –– AdCo ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/adco.png" width = "250px" alt="" />

                                </td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>Adco: Adversarial contrast for efficient learning of unsupervised representations from self-trained negative adversaries</b> <br/>
                                    <font color="blue">Xiao Wang*</font>, Qianjiang Hu*, Wei Hu, Guo-Jun Qi<br/>
                                    <i>Conference on Computer Vision and Pattern Recognition (<font color="red">CVPR</font>)</i>, 2021<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://openaccess.thecvf.com/content/CVPR2021/papers/Hu_AdCo_Adversarial_Contrast_for_Efficient_Learning_of_Unsupervised_Representations_From_CVPR_2021_paper.pdf" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/maple-research-lab/adco" target="_blank" rel="noopener">
                                    <i class="fas fa-code mr-1"></i>Code
                                    </a>
                                    <br/>
                                    <font color="#3399FF">[Representation learning]</font>
                                </td>
                              </tr>
                            </table>

<!-- 
                            <! –– GNN Dove ––>

                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px"><img class="image" src="image/gnndove.jpeg" width = "250px" alt="" /></td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>Protein Docking Model Evaluation by Graph Neural Networks</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Sean T Flannery, Daisuke Kihara <br/>
                                    <i>Frontiers in Molecular Biosciences</i>, 2021<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.frontiersin.org/articles/10.3389/fmolb.2021.647915/full" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/GNN_DOVE" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>
                                        code
                                    </a>
                                    <br/>
                                </td>
                              </tr>
                            </table>

                            <! –– DOVE ––>
                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px"><img class="image" src="image/dove2.jpeg" width = "250px" alt="" /></td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>Protein Docking Model Evaluation by 3D Deep Convolutional Neural Network</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Genki Terashi, Charles W Christoffer, Mengmeng Zhu, Daisuke Kihara <br/>
                                    <i>Bioinformatics</i> , 2019<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://academic.oup.com/bioinformatics/article/36/7/2113/5634144" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/DOVE" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>
                                        code
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://kiharalab.org/dove/" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>Server
                                        </a>
                                    <br/>
                                </td>
                              </tr>
                            </table> -->

<!--                        <table>-->
<!--                            <tr>-->
<!--                                <td align="left" width = "50px"></td>-->
<!--                                <td align="left" width = "350px"><img class="image" src="image/ocfinder.png" width = "250px" alt="" /></td>-->
<!--                                <td align="left" bgcolor="" width = "600px" >-->
<!--                                    <b>OC_Finder: Osteoclast Segmentation, Counting, and Classification Using Watershed and Deep Learning</b> <br/>-->
<!--                                     <font color="blue">Xiao Wang*</font>, Mizuho Kittaka*, Yilin He, Yiwei Zhang, Yasuyoshi Ueki, Daisuke Kihara <br/>-->
<!--                                    <i>Frontiers in Bioinformatics</i> , 2022<br/>-->
<!--                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fbinf.2022.819570" target="_blank" rel="noopener">-->
<!--                                        <i class="fas fa-file-pdf mr-1"></i> Paper-->
<!--                                    </a>-->
<!--                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/kiharalab/OC_Finder" target="_blank" rel="noopener">-->
<!--                                        <i class="fas fa-code mr-1"></i>-->
<!--                                        code-->
<!--                                    </a>-->
<!--                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://bit.ly/OC_Finder" target="_blank" rel="noopener">-->
<!--                                        <i class="fas fa-code mr-1"></i>-->
<!--                                        Colab-->
<!--                                    </a>-->
<!--                                    <br/>-->
<!--                                </td>-->
<!--                              </tr>-->
<!--                            </table>-->

                            <!-- <h4>Representation learning</h4> -->


                            


                           
                           



                            




                            

                <h4>2020</h4>  
                            <! –– EnAET ––>
                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px"><img class="image" src="image/enaet.jpeg" width = "250px" alt="" /></td>
                                <td align="left" bgcolor="" width = "600px" >
                                    <b>EnAET: Self-Trained Ensemble AutoEncoding Transformations for Semi-Supervised Learning</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Daisuke Kihara, Jiebo Luo, Guo-Jun Qi<br/>
                                    <i>IEEE Transactions on Image Processing (<font color="red">IEEE TIP</font>)</i> , 2020<br/>
                                    
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1109/TIP.2020.3044220" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/maple-research-lab/EnAET" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>
                                        code
                                    </a>
                                    <br/>
                                    <b>Learning generalized transformation equivariant representations via autoencoding transformations</b> <br/>
                                    Guo-Jun Qi, Liheng Zhang, Feng Lin, <font color="blue">Xiao Wang</font> <br/>
                                    <i>IEEE Transactions on Pattern Analysis and Machine Intelligence (<font color="red">IEEE T-PAMI</font>)</i>, 2020<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1109/TPAMI.2020.3029801" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/maple-research-lab/AET" target="_blank" rel="noopener">
                                        <i class="fas fa-code mr-1"></i>
                                        code
                                    </a>
                                    <br/>
                                    <font color="#3399FF">[Representation learning]</font>
                                </td>
                              </tr>
                            </table>

                          

                           

<!-- 
                    <h4>Intelligent Transportation</h4>
 

                             <! –– Capture ––>
                            <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/capture.jpeg" width = "250px" alt="" />
                                </td>
                                <td align="left" bgcolor="" width = "600px" >

                                    <b>Capturing Car-Following Behaviors by Deep Learning</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Rui Jiang, Li-Li, Yilun Lin, Xinhu Zheng, Fei-Yue Wang<br/>
                                    <i>IEEE Transactions on Intelligent Transportation Systems (IEEE-T-ITS)</i>, 2017<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://ieeexplore.ieee.org/abstract/document/7970189" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>
                                    <br>
                                    <font color="magenta">Nominated for George N. Saridis Best Transactions Paper Award.</font>

                                </td>
                              </tr>
                            </table>

                         <table>
                            <tr>
                                <td align="left" width = "50px"></td>
                                <td align="left" width = "350px">
                                    <img class="image" src="image/longmemory.jpg" width = "250px" alt="" />
                                </td>
                                <td align="left" bgcolor="" width = "600px" >

                                    <b>Long memory is important: A test study on deep-learning based car-following model</b> <br/>
                                    <font color="blue">Xiao Wang</font>, Rui Jiang,Li-Li, Yilun Lin, Fei-Yue Wang<br/>
                                    <i>Physica A: Statistical Mechanics and its Applications</i>, 2019<br/>
                                    <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.physa.2018.09.136" target="_blank" rel="noopener">
                                        <i class="fas fa-file-pdf mr-1"></i> Paper
                                    </a>




                                </td>
                              </tr>
                            </table> -->



                         <h3>Open Source Projects</h3>

  <table>
    <! –– SLOT ––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>SLOT</b>: SLOT is an Sample-specific Language Model Optimization algorithm at Test-time
&nbsp;<a href="https://github.com/maple-research-lab/SLOT"><img alt="GitHub stars" src="https://img.shields.io/github/stars/maple-research-lab/SLOT?style=social"></a>
                                    </li>

                                </td>
                              </tr>
    <! –– HiCFuondation––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>HiCFoundation</b>:HiCFoundation is a generalizable Hi-C foundation model for chromatin architecture, single-cell and multi-omics analysis across species.
&nbsp;<a href="https://github.com/Noble-Lab/HiCFoundation"><img alt="GitHub stars" src="https://img.shields.io/github/stars/Noble-Lab/HiCFoundation?style=social"></a>
                                    </li>

                                </td>
                              </tr>


    <! –– CryoNeRF––>

                              <tr>
                                  <td align="left" bgcolor="" width = "1000px" >
                                      <li> <b>CryoNeRF</b>: CryoNeRF is a computational tool for homogeneous and heterogeneous (conformational and compositional) cryo-EM reconstruction in Euclidean 3D space.
  &nbsp;<a href="https://github.com/UNITES-Lab/CryoNeRF"><img alt="GitHub stars" src="https://img.shields.io/github/stars/UNITES-Lab/CryoNeRF?style=social"></a>
                                      </li>
  
                                  </td>
                                </tr>

    
    <! –– FlowTS––>

                                <tr>
                                    <td align="left" bgcolor="" width = "1000px" >
                                        <li> <b>FlowTS</b>: FlowTS is a rectified flow-based time series generation algorithm.
    &nbsp;<a href="https://github.com/UNITES-Lab/FlowTS"><img alt="GitHub stars" src="https://img.shields.io/github/stars/UNITES-Lab/FlowTS?style=social"></a>
                                        </li>
    
                                    </td>
                                  </tr>

    <! –– DiffModeler––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>DiffModeler</b>:DiffModeler is a computational tool using diffusion model to automatically build full protein complex structure by taking native/AF2 single-chain structure as input.
&nbsp;<a href="https://github.com/kiharalab/DiffModeler"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/DiffModeler?style=social"></a>
                                    </li>

                                </td>
                              </tr>
        <! –– DeepMainMast––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>DeepMainMast</b>: DeepMainMast is a computational tool using deep learning to automatically build full protein complex structure from cryo-EM map.
&nbsp;<a href="https://github.com/kiharalab/DeepMainMast"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/DeepMainMast?style=social"></a>
                                    </li>

                                </td>
                              </tr>

     <! –– AdPE––>

                              <tr>
                                  <td align="left" bgcolor="" width = "1000px" >
                                      <li> <b>AdPE</b>: AdPE is a MIM based self-supervised learning method with adversarial embeddings.
  &nbsp;<a href="https://github.com/maple-research-lab/AdPE"><img alt="GitHub stars" src="https://img.shields.io/github/stars/maple-research-lab/AdPE?style=social"></a>
                                      </li>
  
                                  </td>
                                </tr>
    
  <! –– CryoREAD––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>CryoREAD</b>: Cryo_READ is a computational tool using deep learning to automatically build full DNA/RNA atomic structure from cryo-EM map.
&nbsp;<a href="https://github.com/kiharalab/CryoREAD"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/CryoREAD?style=social"></a>
                                    </li>

                                </td>
                              </tr>
       <! –– DAQ-Refine––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>DAQ_Refine</b>: DAQ_Refine is a protein structure refinement tool by DAQ-score and ColabFold.
&nbsp;<a href="https://github.com/kiharalab/DAQ-Refine"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/DAQ-Refine?style=social"></a>
                                    </li>

                                </td>
                              </tr>
                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>Asym Siam</b>: Asym-Siam experimentally verified the importance of asymmetry for Siamese Representation Learning with
obvious improvement.
&nbsp<a href="https://github.com/facebookresearch/asym-siam"><img alt="GitHub stars" src="https://img.shields.io/github/stars/facebookresearch/asym-siam?style=social"></a>
                                    </li>

                                </td>
                              </tr>

     <! –– caco ––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>CaCo</b>: CaCo is a state-of-the-art cooperative-adversarial contrastive learning method where both positive and negative
samples are directly learnable.
&nbsp;<a href="https://github.com/maple-research-lab/caco"><img alt="GitHub stars" src="https://img.shields.io/github/stars/maple-research-lab/caco?style=social"></a>
                                    </li>

                                </td>
                              </tr>

                            <! –– DAQ ––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>DAQ</b>: DAQ is a software accesses the quality of protein models built from cryo-Electron Microscopy (EM) maps.
&nbsp;<a href="https://github.com/kiharalab/DAQ"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/DAQ?style=social"></a>
                                    </li>

                                </td>
                              </tr>

                           <! –– OC_Finder acceptance––>

                        <tr>
                           <td align="left" bgcolor="" width = "1000px" >
                               <li> <b>OC_Finder</b>: OC_Finder is a computational tool using deep learning for fully automated osteoclast segmentation, classification,
and counting.
                           <a href="https://github.com/kiharalab/OC_Finder"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/OC_Finder?style=social"></a>
                             </li>
                           </td>
                            </tr>


      <! –– CoSeg acceptance––>

                        <tr>
                           <td align="left" bgcolor="" width = "1000px" >
                               <li> <b>CoSeg</b>: CoSeg is a self-supervised learning-based event boundary detection method.
                           <a href="https://github.com/wang3702/CoSeg"><img alt="GitHub stars" src="https://img.shields.io/github/stars/wang3702/CoSeg?style=social"></a>
                             </li>
                           </td>
                            </tr>

                          <! ––GNN DOVE acceptance––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li>
                                        <b>CLSA</b>: CLSA is a general contrastive learning framework by introducing the information from stronger augmentation.
                                        <a href="https://github.com/maple-research-lab/CLSA"><img alt="GitHub stars" src="https://img.shields.io/github/stars/maple-research-lab/CLSA?style=social"></a>
                                    </li>
                                </td>
                            </tr>




                            <! –– AdCo acceptance––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li> <b>AdCo</b>: AdCo is an algorithm for effective self-supervised learning through adversarial training of negative examples.
                                <a href="https://github.com/maple-research-lab/AdCo"><img alt="GitHub stars" src="https://img.shields.io/github/stars/maple-research-lab/AdCo?style=social"></a>
                                    </li>

                                </td>
                              </tr>

                          <! –– GNN DOVE acceptance––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                 <li> <b>GNN DOVE</b>:  GNN_DOVE is a software can evaluate the quality of protein-docking models using graph neural networks by
reformulating protein structures as graphs.

                                        <a href="https://github.com/kiharalab/GNN_DOVE"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/GNN_DOVE?style=social"></a>

                                </td>

                            </tr>


                            <tr>
                               <td align="left" bgcolor="" width = "1000px" >
                                   <li>
                                   <b>Attention_AD</b>: Attention_AD is a software that can distinguish active and inactive peptides for gene expression using Long
Short Term Memory (LSTM).
                                    <a href="https://github.com/kiharalab/Attention_AD"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/attention_ad?style=social"></a>
                                   </li>
                               </td>
                            </tr>


                        <!–– Emap2secplus acceptance––>

                           <tr>
                               <td align="left" bgcolor="" width = "1000px" >
                                 <li> <b>Emap2sec+</b>: Emap2sec+ is a software detects local structure information of proteins and DNA/RNA in cryo-EM maps using
deep learning.
                                    <a href="https://github.com/kiharalab/emap2secplus"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/emap2secplus?style=social"></a>
                                 </li>

                               </td>



                           </tr>





                            <!–– EnAET acceptance––>

                            <tr>
                                <td align="left" bgcolor="" width = "1000px" >
                                    <li>
                                      <b>EnAET</b>: EnAET is a software that benefits semi-supervised learning via self-trained ensemble auto-encoding transformations.
                                        <a href="https://github.com/maple-research-lab/EnAET"><img alt="GitHub stars" src="https://img.shields.io/github/stars/maple-research-lab/EnAET?style=social"></a>  <a href="https://github.com/wang3702/EnAET"><img alt="GitHub stars" src="https://img.shields.io/github/stars/wang3702/EnAET?style=social"></a>
                                    </li>

                                </td>
                            </tr>



                        <!–– DOVE acceptance––>

                            <tr>
                               <td align="left" bgcolor="" width = "1000px" >
                                   <li>
                                   <b>DOVE</b>: DOVE is a software can evaluate the quality of protein-docking models using 3D neural networks.
                                    <a href="https://github.com/kiharalab/DOVE"><img alt="GitHub stars" src="https://img.shields.io/github/stars/kiharalab/dove?style=social"></a>
                                   </li>
                               </td>
                            </tr>
                        </table>

<!--                 <h3>Professional Service</h3>-->
<!--                        <table>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Associate Editor (AE) of IEEE Transactions on Intelligent Vehicles (T-IV).</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of NSF Graduate Research Fellowships Program (GRFP).</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of IEEE Transactions on Pattern Analysis and Machine Intelligence</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of IEEE Transactions on Image Processing</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of Pattern Recognition</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of Bioinformatics</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of IEEE Transactions on Intelligent Transportation Systems</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of IEEE Transactions on Multimedia</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of ACM Transactions on Knowledge Discovery from Data</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of IEEE Intelligent Systems</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of Frontiers in Bioinformatics</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of Conference on Neural Information Processing Systems (NeurIPS)</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of Conference on Computer Vision and Pattern Recognition (CVPR)</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of International Conference on Computer Vision (ICCV)</li>-->
<!--                               </td>-->
<!--                            </tr>-->
<!--                            <tr>-->
<!--                               <td align="left" bgcolor="" width = "1000px" >-->
<!--                                    <li>Reviewer of European Conference on Computer Vision (ECCV)</li>-->
<!--                               </td>-->
<!--                            </tr>-->

<!--                        </table>-->





                    </div>
                </div>
            </div>
        </div>

        <div id="footer">

        </div>



  <div class="container">
    <footer class="site-footer">


  <p class="powered-by">

    © 2021 - 2025 · Xiao Wang




    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>

  </p>
</footer>

  </div>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-88566694-1', 'auto');
  ga('send', 'pageview');

</script>        


<script src="js/jquery.min.js"></script>
<script src="js/tether.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-89924514-1', 'auto');
  ga('send', 'pageview');

</script>

<!-- previous track website down, use new one instead -->
<!-- <script type="text/javascript" src="//rf.revolvermaps.com/0/0/8.js?i=5no0il9epuq&amp;m=0&amp;c=ff0000&amp;cr1=ffffff&amp;sx=0" async="async"></script> -->
<!-- <script type="text/javascript" id="clstr_globe" src="//clustrmaps.com/globe.js?d=7e1XVoQeWUvHmwfThy4G2T2QUHl0lWh59stCFmeN4XU"></script> -->
<script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=tt&d=7e1XVoQeWUvHmwfThy4G2T2QUHl0lWh59stCFmeN4XU'></script>
    </body>
</html>
